### ISSUE TITLE
How can i reset dropdown commandHandler Data?

### ISSUE THREAD

**Issue by jypblue** [element-plus/element-plus#445]
``` 
<el-dropdown @command="commandHandler">
        <div class="flex flex-align-center" ref="">
          <el-badge is-dot class="profile-status__dot" :type="receptionState.type"></el-badge>
          <div>{{ receptionState.label }} <i class="el-icon-arrow-right"></i></div>
        </div>
        <template #dropdown>
          <el-dropdown-menu>
            <el-dropdown-item :command="state" v-for="(state, i) in receptionStateList" :key="i">
              <div class="flex flex-align-center">
                <el-badge is-dot class="profile-status__dot" :type="state.type"></el-badge>
                <div>{{ state.label }}</div>
              </div>
            </el-dropdown-item>
          </el-dropdown-menu>
        </template>
      </el-dropdown>
```
```
const receptionState: ReceptionState = reactive({
      label: "在线",
      value: "ONLINE",
      type: "success"
    });
    const commandHandler = (event: ReceptionState) => {
      console.log("setup event", event);

      receptionState.label = event.label;
      receptionState.value = event.value;
      receptionState.type = event.type;
      console.log(receptionState);
    };

    return {
      receptionState,
      commandHandler
    };

```

i can get the commandHandler data and reset it success, but the template it's not work, it's still default value

**Comment by bastarder** [comment]
i test that, everything is correct, can you give me online example ?  such as `codepen`, or more information
```
<template>
  <el-dropdown @command="commandHandler">
    <div ref="" class="flex flex-align-center">
      <el-badge is-dot class="profile-status__dot" :type="receptionState.type" />
      <div>{{ receptionState.label }} <i class="el-icon-arrow-right"></i></div>
    </div>
    <template #dropdown>
      <el-dropdown-menu>
        <el-dropdown-item v-for="(state, i) in receptionStateList" :key="i" :command="state">
          <div class="flex flex-align-center">
            <el-badge is-dot class="profile-status__dot" :type="state.type" />
            <div>{{ state.label }}</div>
          </div>
        </el-dropdown-item>
      </el-dropdown-menu>
    </template>
  </el-dropdown>
</template>

<script>

import { reactive,ref } from 'vue'
export default {
  setup() {

    const receptionStateList = ref([
      { label: '1', value: '1', type: 'success' },
      { label: '2', value: '2', type: 'error' },
      { label: '3', value: '3', type: 'success' },
    ])
    const receptionState = reactive({
      label: '在线',
      value: 'ONLINE',
      type: 'success',
    })
    const commandHandler = event => {
      console.log('setup event', event)

      receptionState.label = event.label
      receptionState.value = event.value
      receptionState.type = event.type
      console.log(receptionState)
    }

    return {
      receptionState,
      commandHandler,
      receptionStateList,
    }
  },
}
</script>

```

**Comment by jypblue** [comment]
thx

**Comment by jypblue** [comment]
![image](https://user-images.githubusercontent.com/5317786/96963543-62d03c80-153b-11eb-89b7-18df18695aee.png)
version is: `"element-plus": "1.0.1-alpha.4",`

**Comment by jypblue** [comment]
fixed by version: "element-plus@1.0.1-alpha.10"
